### PR TITLE
savings-card-dashboard

### LINKED ISSUE #83: [FEATURE] Visualize savings in the Dashboard page
### What?

For now, the savings of the users are not being displayed in the Dashboard. 

<p align="center">
image
</p>

### Why?

The users are making their transaction as Savings, but they are never visualized. 

### How?

Create a section on the dashboard for the savings. 

<p align="center">
image
</p>

Use the appropriate icon to match the overall design: https://icones.js.org/collection/all

### PR DESCRIPTION
**Pull request by revanthnetha**
Fixes #83

Approach:

1. Created a new Component of savings.tsx which renders the savings card on dashboard as expected.
2. Changed the width in incomes, expenses, savings for clean layout.

image

### CODE REVIEW

**Comment by revanthnetha** [comment]
Check this @manuelalferez . Feel free to discuss if any changes needed :)

File: src/app/dashboard/components/total/savings.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                stroke-linecap="round"
+                stroke-width="1.5"
+                d="M10 22h3c3.771 0 5.657 0 6.828-1.172c.809-.808 1.06-1.956 1.137-3.828m0-6c-.078-1.872-.328-3.02-1.137-3.828C18.657 6 16.771 6 13 6h-3C6.229 6 4.343 6 3.172 7.172S2 10.229 2 14s0 5.657 1.172 6.828c.653.654 1.528.943 2.828 1.07M6 6l3.735-2.477a3.24 3.24 0 0 1 3.53 0L17 6"
+              />
+            </g>
+          </svg>
+          Savings
+        </CardDescription>
+        <CardTitle className="text-green-500 text-lg md:text-xl font-mono tabular-nums">
+          {savings !== 0 && "-"}
```
[the remark this answers is not in the document]
Comment (manuelalferez) [reply src/app/dashboard/components/total/savings.tsx:58]: Small fix

File: src/app/dashboard/components/dashboard.tsx
Comment (manuelalferez) [inline src/app/dashboard/components/dashboard.tsx:1]: Can you please check why is not being responsive? 

https://github.com/user-attachments/assets/01437b41-e0dd-4d3b-be6b-fe9ddee7c3f4

**Comment by manuelalferez** [comment]
Just small improvements. Great job @revanthnetha, omg you work super fast! Very clean code btw 👌

**Comment by revanthnetha** [comment]
Sure! It's actually breaking at endpoints. I'll make it responsive.

**Comment by revanthnetha** [comment]
Could you check the changes @manuelalferez .

**Comment by manuelalferez** [comment]
Nice, much better! Btw, the savings shouldn't change with the filter of above: 

image

If you have 10K in saving, those 10K should be fixed. 

Also, please remove the savings information on the left, it is redundant information. 

Thanks!

**Comment by revanthnetha** [comment]
Removed Savings info on left and shown consistent value in savings. @manuelalferez

**Comment by manuelalferez** [comment]
Awesome @revanthnetha! Merging the PR now. Thanks for the feature

**Pull request merged**